libenv/unix_iface.c

```diff
@@ -657,9 +661,15 @@ static void FindV6InterfacesInfo(EvalContext *ctx)
 
                 if ((IsIPV6Address(ip->name)) && ((strcmp(ip->name, "::1") != 0)))
                 {
+                    char prefixed_ip[PREFIXED_IPV6_LEN] = {0};
+                    size_t ret;
                     Log(LOG_LEVEL_VERBOSE, "Found IPv6 address %s", ip->name);
                     EvalContextAddIpAddress(ctx, ip->name, NULL); // interface unknown
                     EvalContextClassPutHard(ctx, ip->name, "inventory,attribute_name=none,source=agent");
+
+                    ret = snprintf(prefixed_ip, PREFIXED_IPV6_LEN, "ipv6_%s", ip->name);
```

Comment:
`xsnprintf()` is for you! It combines `snprintf()` and `assert()`!

Reply:
Ha, nice! I thought we only have these variants for functions that allocate memory. :)

Reply:
Fixed.